assemblyline/odm/models/submission.py

```diff
@@ -44,46 +49,47 @@ class ServiceSelection(odm.Model):
 }
 
 
-@odm.model(index=True, store=False, description="Submission Parameters")
+@odm.model(index=True, store=False, description="""Specifies the parameters set at the time of submission, such as whether to perform a deep scan, if the submission should generate an alert, and any specific service selections or settings.
+""")
 class SubmissionParams(odm.Model):
     classification = odm.Classification(default=Classification.UNRESTRICTED,
-                                        description="Original classification of the submission")
-    deep_scan = odm.Boolean(default=False, description="Should a deep scan be performed?")
-    description = odm.Text(store=True, copyto="__text__", description="Description of the submission")
-    generate_alert = odm.Boolean(default=False, description="Should this submission generate an alert?")
-    groups = odm.List(odm.Keyword(), default=[], description="List of groups related to this scan")
-    ignore_cache = odm.Boolean(default=False, description="Ignore the cached service results?")
+                                        description="The initial security classification for the submission, indicating its confidentiality.")
+    deep_scan = odm.Boolean(default=False, description="**TODO**: **Original**:Should a deep scan be performed?  **Generated**:Specifies whether a more intensive scan should be conducted on the submission.")
+    description = odm.Text(store=True, copyto="__text__", description="A user-provided description or notes regarding the submission.")
```

Comment:
Those are not notes, this is the "Description" which will appear in the "Description" column of the Submission list page. Notes should go in the metadata.
I think it would be analogous to talking about the title of a book, would notes are inside, in the appendix.

Reply:
I see your point.  Would that be better?
> A user-provided narrative that outlines the content and purpose of the submission.

Reply:
Yes, but at the same time, if the user does not provide one, we make one by default (I would need to check if it's in the API, or the client, but I don't specify my description in my personal scripts).

Reply:
Let's drop `user-provided` then.